block-fixups/run.sh

```
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+#
+# Alert when there are fixup or squash commits
+#
+
```

probably worth adding `set -eo pipefail` here to ensure we catch any ambiguous errors

---

Added. Thanks 👍

---

Tested here: 
- Success: https://github.com/futurelearn/test-github-actions/actions/runs/319776756
- Fixup failure: https://github.com/futurelearn/test-github-actions/runs/1286530513?check_suite_focus=true
- Squash failure: https://github.com/futurelearn/test-github-actions/runs/1286532944?check_suite_focus=true